styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -62,6 +70,26 @@ import templateAdviser from './_adviser.html.haml';
   </Story>
 </Preview>
 
+## When to use
+
+Callouts can be used to highlight information within content, for example:
+
+- Important information 
+- Adviser-only information
+- Examples
+- Quotes
+
+Before using a callout, make sure you really need it. Fewer callouts have more of an impact on a piece of content. Too many callouts can confuse and distract users from other important content on a page. 
+
+## Usage
+
+Callouts should come with a title that explains the subject of the information when possible. This will give the users an idea of what the information is about.
+
+### Markup
+
+The title should be used as `h3` no matter where they are on the page. If a callout is the first thing on the page under a `h1`, still put it in the backend as a `h3` and the CMS will make sure it displays correctly.
```

Comment:
The design system probably shouldn't have too much knowledge of Contentful here. I actually think this is a problem with the content model if we're asking people to knowingly put in incorrect headings. Ideally the content model should have a dedicated title field, and the website could work out what the best heading level is based on context.

For now, I'd suggest rather than putting this guidance here we maybe add it as a help field in Contentful?

Reply:
Not sure about this one, as we originally included content designers as the users of the Design System. What are your thoughts @soniaturcotte ?

Reply:
Ah yeah, I'm mostly splitting hairs about where this guidance should _live_. It feels like the wrong level, but that's mostly because the way we solve this is kinda funky via some javascript after the fact rather than solving it within the content model. I reckon we could leave this as is though as it reflects the current state of things.

Reply:
Sounds good to me